200.blacklists/metaports.yaml

```diff
@@ -8,6 +8,9 @@
 
 - { remove: true,   name: autotools }
 
+- { remove: true,   namepat: 'base-.*',  ruleset: void }
+- { remove: true,   namepat: '.*-base',  ruleset: void }
```

Comment:
Too broad and dangerous. For instance, .*-base covers perl:test-base. Why are these needed at all?